src/main/scala/cromwell/engine/backend/CromwellBackend.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.backend
+
+import akka.actor.ActorSystem
+import org.slf4j.LoggerFactory
+import Backend.BackendyString
+
+/**
+  * Provides a global singleton access to the instantiated backend. This isn't bulletproof and can lead to
+  * exceptions being thrown but the way the system utilizes it it shouldn't be a problem (famous last words).
+  * So far the only real oddities I've seen are from the unit tests when they were trying to go straight to the
+  * interior of the system.
+  *
+  * Also we expect nearly everything to change about Backend in the next couple of months due to Intel's
+  * incoming contributions so this seems like a good enough for now thing.
+  */
```

Comment:
I think this comment is mostly unnecessary (first 2 sentences and the comment about Intel don't really add any value, I feel) and you left me hanging on the last sentence!

> the only real oddities I've seen are from the unit tests when they were trying to go straight to the interior of the system.

Well, what are the oddities?  How should I be using this in a test?